CefSharp.Wpf/Experimental/ChromiumWebBrowserWithTouchSupport.cs

```diff
@@ -118,38 +182,54 @@ private void OnTouch(TouchEventArgs e)
             }
         }
 
+        #endregion Touch
+
+        #region Stylus Pen
+        //Both touch and stylus pen will Run OnStylusXXX.
+        //We use OnTouchXXX to handle touch, which is faster.
```

Comment:
We use the OnTouchXXX methods which contain more intermediate points to handle the user's touch so that we can track the user's fingers faster.